es/separable_natural_es.py

```diff
@@ -31,12 +57,28 @@ def optimize(func, mu, sigma,
     elif isinstance(rng, int):
         rng = np.random.RandomState(seed=rng)
 
+    mu = mu.copy()
+    sigma = sigma.copy()
     generation = 0
     history_mu = []
     history_sigma = []
     history_pop = []
     history_fitness = []
 
+    mutable_locals = ['rng', 'mu', 'sigma', 'generation', 'history_mu', 'history_sigma', 'history_pop', 'history_fitness']
+
+    if load_existing_checkpoint:
+        state = load_checkpoint()
+        if state:
```

Comment:
If `state` is `None` here, i.e. if there were no checkpoints that can be loaded, I suggest to raise an error. Otherwise checkpoint loading fails silently.